web/src/tippyjs.ts

```diff
@@ -617,6 +617,12 @@ export function initialize(): void {
             defaultMessage: "This profile field is required.",
         }),
         appendTo: () => document.body,
+        onShow(instance) {
+            if ($(instance.reference).children("label").eq(0).hasClass("required-field")) {
```

Comment:
Thanks for pointing out this bug @tnmkr.

I think you can change the `target` to `.custom-user-field-label-wrapper .required-field` instead of doing this, right?

Reply:
Yes! that is cleaner. But changing the target has (a minor) side-effect of tooltip not being triggered when hovering the asterisk symbol beside empty required fields.

Reply:
Oh, that's right. In that case, we can move `{{#if field.required}}required-field{{/if}}` block from `label` to `<span class="custom-user-field-label-wrapper">` in `custom_user_profile_field.hbs` and change the `target` to `.custom-user-field-label-wrapper.required-field`.

Does that sound good?

Reply:
Yup! Updated the PR @Vector73 

I used `required-field-wrapper` instead of `required-field` as I felt that reflected the purpose of that class better. Please take a look.

Reply:
LGTM. @timabbott do you want to take a look?